packages/core/src/connectivity/scp-cf/destination/circuit-breaker.spec.ts

```diff
@@ -0,0 +1,86 @@
+import nock from 'nock';
```

Comment:
It makes sense to have a centralised location for testing circuit breaker instead of moving to multiple locations.
However, this also violates our spec file naming rule, because there is no `circuit-breaker.ts` ;)
I'm fine with it as the exception.

Reply:
Agreed, we also have this in some other places, maybe we should write it down somewhere.